revolv/payments/models.py

```diff
@@ -78,50 +246,64 @@ def donations(self, user=None, project=None, queryset=None):
         :return: Returns all payments that are not repayments or reinvestments
                  associated with this user.
         """
-        return self.payments(user, project, queryset).exclude(payment_instrument_type__name=INSTRUMENT_REINVESTMENT).exclude(payment_instrument_type__name=INSTRUMENT_REPAYMENT)
+        return self.payments(user, project, queryset).exclude(payment_type__name=PAYTYPE_REINVESTMENT)
 
     def reinvestments(self, user=None, project=None, queryset=None):
         """
         :return: Returns all reinvestment payments that are associated with
                  this user.
         """
-        return self.payments(user, project, queryset).filter(payment_instrument_type__name=INSTRUMENT_REINVESTMENT)
+        return self.payments(user, project, queryset).filter(payment_type__name=PAYTYPE_REINVESTMENT)
 
-    def repayments(self, user=None, project=None, queryset=None):
+    def repayments(self, user, admin=None, project=None, queryset=None):
         """
         :return: Returns all the repayments that are associated with this
                  user.
         """
-        return self.payments(user, project, queryset).filter(payment_instrument_type__name=INSTRUMENT_REPAYMENT)
-
-    def all_donations(self, queryset=None):
-        """
-        :return: Returns all the donations made on the application. Useful helper for distinct
-                 donors.
-        """
-        return self.payments(queryset=queryset).exclude(payment_instrument_type__name=INSTRUMENT_REINVESTMENT).exclude(payment_instrument_type__name=INSTRUMENT_REPAYMENT)
+        return AdminRepayment.objects.filter(revolvuserprofiles__in=user, projects__in=project).distinct()
 
-    def total_distinct_donors(self, queryset=None):
+    def total_distinct_organic_donors(self, queryset=None):
         """
-        :return: The total number of donors that have ever donated to a RE-volv
-            project. Useful for displaying stats on the homepage.
+        :return:
+            The total number of organic donors that have ever donated to a
+            RE-volv project. Useful for displaying stats on the homepage.
+            Organic donors are donors that have made a non-reinvestment type
```

Comment:
Organic donors here DON'T include donations via check which have a user=None, right? I feel like that would be good to note here